### PR TITLE
Luis Martinez

### PR DESCRIPTION
**Pull request by luiscmartinez**

### CODE REVIEW

File: fizzbuzz/fizzbuzz.c [hunk 1]
```diff
@@ -10,7 +10,25 @@
 */
 int fizzbuzz(int n)
 {
-
+int i;
+int counter = 0;
+  for( i = 1; i <= n; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {
```
Comment (blauente) [inline fizzbuzz/fizzbuzz.c]: Great job putting this conditional first! You can also check `if (i % 15 == 0)`.

File: strings/strings.c [hunk 2]
```diff
@@ -9,7 +9,13 @@
 */
 int string_length(char s[])
 {
-
+  int i = 0;
+  char string = s[i];
+  while (string != '\0') {
+    i++;
```
Comment (blauente) [inline strings/strings.c]: Good job iterating this first.

File: strings/strings.c [hunk 3]
```diff
@@ -20,7 +26,22 @@ int string_length(char s[])
...
+  //   rv[length] = '\0';
+  //   return rv;
+  int len = strlen(s);
+  int count = 0;
+  for (int i = len -1; i >= 0; i--) {
+    *(rv + count) = *(s + i);
+    count++;
+  }
+  *(rv + count) = '\0';
+  return rv;
```
Comment (blauente) [inline strings/strings.c]: Are you still getting a segmentation fault on this file?

Comment (luiscmartinez) [reply strings/strings.c]: yesssir i am @blauente

Comment (luiscmartinez) [reply strings/strings.c]: no siirr cx

**Pull request closed without merging**